ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/discovery/PeerDiscoveryAgent.java

```diff
@@ -80,7 +72,6 @@
   /* The keypair used to sign messages. */
   protected final SECP256K1.KeyPair keyPair;
   private final BytesValue id;
-  private final PeerTable peerTable;
```

Comment:
`PeerTable` isn't used in `PeerDiscoveryAgent` - moved it into the `PeerDiscoveryController` where it is used.